examples/addr2line.rs

```diff
@@ -4,18 +4,30 @@ extern crate fallible_iterator;
 extern crate gimli;
 extern crate memmap;
 extern crate object;
+extern crate typed_arena;
 
 use std::borrow::Cow;
 use std::fs::File;
 use std::io::{BufRead, Lines, StdinLock, Write};
 use std::path::Path;
+use std::result;
 
 use clap::{App, Arg, Values};
 use fallible_iterator::FallibleIterator;
-use object::Object;
+use object::{Object, ObjectSection};
+use typed_arena::Arena;
 
 use addr2line::{Context, Location};
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Error {
```

Comment:
Let's leave this out for now. Better error handling would be fine, but you're not actually creating any instances of this yet (`load_file_section` always returns `Ok`).

Reply:
That's true, but `load_file_section` needs to return a Result. What should the error type be?

Reply:
It can be `()`.